Tools/CheckCmdletXmlDocs.psm1

```diff
@@ -147,8 +149,52 @@ function Check-CmdletDoc() {
     Write-Host
 }
 
+# Get the cmdlets explicitly named as a subset of all Google Cloud cmdlets.
+function GetCmdletsByName ($cmdletNames, $allCmdlets) {
+    PrintElementsNotFound $cmdletNames $allCmdlets.Name "`nThe following cmdlets you named were not found:"
+    return @($allCmdlets | where { $cmdletNames -contains $_.Name })
```

Comment:
You can change the where statement to be "where Name -in $cmdletNames". I think it reads better when the property from the piped object comes first. Also, Where-Object has a Parameter Set for each comparison operator, allowing you to avoid using a script block and the reference to $_.